fe/fe-core/src/main/java/com/starrocks/catalog/PrimitiveType.java

```diff
@@ -377,6 +377,11 @@ public int getTypeSize() {
                 // 1MB
                 typeSize = 1024 * 1024;
                 break;
+            case JSON:
+                typeSize = 1024;
```

Comment:
1kb is too small for json type?

Reply:
As SSB flat dataset, convert the table row to JSON costs about 800+ bytes per JSON field.

Reply:
I don't konw, but th user's data isn't like SSB only int and short-string, maybe we need remeber to adjust the size

Reply:
Actually in normal case the statistic executor don't need to use this typeSize to estimate the row-size, it use `char_length` function to touch the the real data.